internal/vercheck/vercheck.go

```diff
@@ -8,36 +8,50 @@ import (
 	"os/exec"
 	"path/filepath"
 
+	"github.com/fatih/color"
 	"github.com/pkg/errors"
+	"go.jetpack.io/devbox/internal/build"
+	"go.jetpack.io/devbox/internal/debug"
 	"golang.org/x/mod/semver"
 
 	"go.jetpack.io/devbox/internal/boxcli/usererr"
 	"go.jetpack.io/devbox/internal/env"
-	"go.jetpack.io/devbox/internal/ux"
 	"go.jetpack.io/devbox/internal/xdg"
 )
 
 // Keep this in-sync with latest version in launch.sh. If this version is newer
-// Than the version in launch.sh, we'll print a warning.
-const expectedLauncherVersion = "v0.1.0"
+// than the version in launch.sh, we'll print a notice.
+// TODO savil: bump to new version after https://github.com/jetpack-io/axiom/pull/3150 lands
+const expectedLauncherVersion = "v0.2.0"
 
 // currentChannel represents the CLI update channel. For now, we set it to a
 // constant, but we'll expand this to be a variable in the future.
 const currentChannel = "stable"
 
-func CheckLauncherVersion(w io.Writer) {
+// currentBinaryVersion is the version of the binary that is currently running.
+// We use this variable so we can mock it in tests.
+var currentBinaryVersion = build.Version
+
+// CheckVersion checks the launcher and binary versions and prints a notice if
+// they are out of date.
+func CheckVersion(w io.Writer) {
+
 	if env.IsDevboxCloud() {
 		return
 	}
 
-	if isNewLauncherAvailable() {
-		ux.Fwarning(
-			w,
-			"newer launcher version %s is available (current = %s), please update "+
-				"using `devbox version update`\n",
-			expectedLauncherVersion,
-			currentLauncherVersion(),
-		)
+	launcherNotice := launcherVersionNotice()
+	if launcherNotice != "" {
+		color.New(color.FgYellow).Fprintf(w, launcherNotice)
```

Comment:
`ux.Fwarning` ?

Reply:
or maybe a new `ux.FNotice` ?

Reply:
I first added `ux.Fwarning` but then backtracked since its not a _warning_ to our users, and yeah, more like a Notice.

Reply:
I'm going to punt on this for now to limit size